plat/rockchip/common/aarch64/plat_helpers.S

```diff
@@ -35,6 +35,9 @@
 #include <cortex_a72.h>
 #include <plat_private.h>
 #include <platform_def.h>
+#if NEED_PMU_MACROS_S
+#include <plat_pmu_macros.S>
+#endif
```

Comment:
The codes are for all of socs, we consider that maybe some socs is not need，so we add this macros. but we consider it is not necessary now.
Okay remove it.